### PR TITLE
some dependencies updated (Not key ones like numpy or matplotlib)

### LINKED ISSUE #253: mpi4py is not in requirements
mpi4py is used in pw_protocol_mpirun.py (and in several plugins) but it is not installed by scipion. That is, the user needs to install it by hand. I suggest to add in "requirements.txt"     mpi4py>=3.0.3

### PR DESCRIPTION
**Pull request by pconesa**
Timer class improvement when used in with clause.

Closes #253

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,8 +1,9 @@
-bibtexparser<=1.2.0
-psutil<=5.7.0
-configparser<=5.0.0
-matplotlib==3.0.3; python_version == '3.5'
-matplotlib==3.2.2; python_version >= '3.6'
-numpy<=1.18.4
-pillow<=7.1.2
-requests<=2.25.1
\ No newline at end of file
+bibtexparser<=1.2.0 # up to date on 07-2021
```
Comment (delarosatrevin) [inline requirements.txt:1]: Are you already in July 2021? ;)

**Comment by pconesa** [comment]
I'm predicting the time this PR will be accepted.

🤦‍♂️

I was even hesitating about the year..

File: requirements.txt (hunk 1 above)
Comment (azazellochg) [reply requirements.txt:1]: Almost there...

**Comment by pconesa** [comment]
Conflicts resolved

**Review by fonsecareyna82 — APPROVED** [review]

**Pull request merged**